chore: add vale linting (#1856)
GetStream/stream-chat-react-native: docusaurus/docs/reactnative/customization/typescript.mdx

};
```

> Additional fields on the defaults, that is `image`, are custom fields used by `stream-chat-react-native` already within the SDK.

Reviewer: ```suggestion
Additional fields on the defaults, for example `image`, are custom fields used by `stream-chat-react-native` already within the SDK.
```

I think i.e. was used incorrectly here
